docker/pravega/entrypoint.sh

```diff
@@ -48,9 +50,23 @@ configure_tier2() {
         done
     fi
     add_system_property "filesystem.root" "${NFS_MOUNT}"
+    ;;
+
+    HDFS)
     add_system_property "hdfs.hdfsUrl" "${HDFS_URL}"
     add_system_property "hdfs.hdfsRoot" "${HDFS_ROOT}"
     add_system_property "hdfs.replication" "${HDFS_REPLICATION}"
+    ;;
+    EXTENDEDS3)
```

Comment:
This was ECS before. Has gone through ECS -> EXTS3 -> EXTENDEDS3 renames. The last consensus was EXTENDEDS3.